### DISCUSSION TITLE
Lag between last frame and first frame when looping

### DISCUSSION THREAD

**Discussion by voidastro4** [mpv-player/mpv#14683] [Q&A]
With loop enabled I have noticed a problem that when videos get to a certain size there is a lag between the last frame and first frame.

Is this a bug or am I doing or not doing something wrong?

If this a bug perhaps it would make sense for mpv to change behavior when loop is enabled to have seamless continuation?

**Comment by Terry-0424** [comment]
have the same problem..

**Reply by voidastro4** [reply]
I found no solutions.
